$OpenBSD: patch-libraw_r_pc_in,v 1.2 2020/08/22 08:21:40 rsadowski Exp $

Do not hardcode -lstdc++ (symbol mismatch with libc++abi and libestdc++).

Index: libraw_r.pc.in
--- libraw_r.pc.in.orig
+++ libraw_r.pc.in
@@ -7,5 +7,5 @@ Name: libraw
 Description: Raw image decoder library (thread-safe)
 Requires: @PACKAGE_REQUIRES@
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lraw_r -lstdc++@PC_OPENMP@
+Libs: -L${libdir} -lraw_r@PC_OPENMP@
 Cflags: -I${includedir}/libraw -I${includedir}
